### PR TITLE
HW1 Part 1, 2, 3

### PR DESCRIPTION
**Pull request by Mg4145**
Finished HW 1

- Completed README for self repo 
- Updated readme fork with two other readmes for proposal and contribution.

### CODE REVIEW

File: projects/mupen64plus.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Mupen64Plus and mupen64plus-ui-python
```
Comment (timkpaine) [inline projects/mupen64plus.md:1]: move to `projects/python/mupen64plus.md`

File: projects/sticks.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Sticks
```
Comment (timkpaine) [inline projects/sticks.md:1]: move to e.g. `projects/python/sticks.md`

**Review by timkpaine — CHANGES_REQUESTED** [review]
undo removal of all other students' markdown files

File: projects/mupen64plus.md (hunk 1 above)
Comment (Mg4145) [reply projects/mupen64plus.md]: Done!

File: projects/sticks.md (hunk 2 above)
Comment (Mg4145) [reply projects/sticks.md]: Done!

**Comment by Mg4145** [comment]
* Undid the readme removals 
* Moved both MD files to a python directory in projects

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**